lib/libutils/isoc/include/malloc.h

```diff
@@ -89,4 +89,7 @@ size_t malloc_get_max_allocated(void);
 size_t malloc_get_allocated(void);
 size_t malloc_get_heap_size(void);
 
+void __malloc_lock(void);
+void __malloc_unlock(void);
```

Comment:
It turned out to not be a problem with `extern struct mutex __mallloc_mu` in `malloc.h`